ui/documents/semantic-ui/templates/records_dashboard/search/ResultsListItem.jsx

```diff
@@ -0,0 +1,2 @@
+import NRDocsResultsListItem from "../../documents/search/ResultsListItem";
```

Comment:
please use alias @js/documents/...

Reply:
@mesemus Yes clear, I need to refactor dashboard first, so this will change anyway and then I will fix it by using the alias.